### PR TITLE
Update mixer.c

### PR DESCRIPTION
**Pull request by KenImhof**
Make 3D center stick symmetrical by applying offset error, Issue #1879

### CODE REVIEW

File: src/main/flight/mixer.c [hunk 1]
```diff
@@ -500,7 +500,7 @@ void mixTable(void)
     // Find min and max throttle based on condition.
     if (feature(FEATURE_3D)) {
         if (!ARMING_FLAG(ARMED)) throttlePrevious = rxConfig()->midrc; // When disarmed set to mid_rc. It always results in positive direction after arming.
-
+        rcCommand[THROTTLE] = rcCommand[THROTTLE] + 57; //Adjust Center Stick Accuracy, Issue #1879
```
Comment (DzikuVx) [inline src/main/flight/mixer.c:503]: There has to be a better way than hardcoded 57 here...

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/flight/mixer.c (hunk 1 above)
Comment (digitalentity) [reply src/main/flight/mixer.c:503]: I second @DzikuVx's comment. A magic number that's only valid for certain configuration is not a good way to fix things.

We probably need to re-think how 3D throttle is handled.

**Comment by digitalentity** [comment]
We need to understand what parameters affect 3D mode and in what way and change that to respect possible assymetric input.

The bug here is that code calculating `rcCommand[THROTTLE]` is not aware of 3D mode.

Suggestion - change rcCommand to float. This way `rcCommand[THROTTLE]` will be in range [0;1] for normal mode and [-1;1] for 3D mode. There will be two mappings in the code - in RC input handling and in ESC output handling. Warning - this will be a massive rework.

**Comment by KenImhof** [comment]
Yes, I believe the root problem is that rcCommand[THROTTLE] is not calculated as a float so the calculated THROTTLE value is not accurate.  Since the 3D critical point is the mid-stick (1500ms) this is just a simple work-around to make the 3D THROTTLE accurate around center stick.  I agree that this does not fix the end-point errors and this will increase the THROTTLE error at the low point.  That is why I am proposing just applying this work-around to the 3D code.

**Comment by digitalentity** [comment]
Closing, this PR is not a proper way to fix the issue.

**Pull request closed without merging**